packages/experimental/kai/project.json

```diff
@@ -2,7 +2,25 @@
   "sourceRoot": "packages/experimental/kai/src",
   "projectType": "application",
   "targets": {
+    "build": {
+      "dependsOn": [
+        "^build",
+        "prebuild"
```

Comment:
I believe @richburdon has a PR which includes prebuild in the default `dependsOn` for build/bundle/serve across the workspace